controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -67,6 +67,33 @@ public ControllerClusterListener(final Host host, final Cluster cluster,
         this.sweepers = Lists.newArrayList(sweepers);
     }
 
+    /**
+     * Get the zookeeper health status.
+     *
+     * @return true if zookeeper is connected.
+     */
+    public boolean isMetadataServiceConnected() {
+        return cluster.isHealthy();
+    }
+
+    /**
+     * Get the sweepers status.
+     *
+     * @return true if all sweepers are ready.
+     */
+    public boolean areAllSweepersReady() {
```

Comment:
Please add code to existing tests to cover these uncovered lines reported by `codecov patch`.

Reply:
Done